src/main/java/organice/model/person/Donor.java

```diff
@@ -50,6 +55,21 @@ public OrganExpiryDate getOrganExpiryDate() {
         return organExpiryDate;
     }
 
+    public ArrayList getProcessingList(Nric nric) {
```

Comment:
Would recommend argument name to be changed from `nric` to `patientNric` for clarity

Reply:
done